### PR TITLE
Service Now Integration - Fixes issue #634

### PR DESCRIPTION
**Pull request by michalis1**
### Fixes issue 634

When running the Nautobot ⟹ ServiceNow job with a site filter set, you expect the job to sync only the specified locations to ServiceNow. 
However, it appears that the job currently loads only the parent locations, not the specified location itself.
This is fixed by initializing the locations list with the user-selected location and then loading the parent locations of that selection.

### CODE REVIEW

File: changes/634.fixed [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ServiceNow Integration
```
Comment (jdrew82) [inline changes/634.fixed:1]: Please rephrase this to be a bit more concise. This is the changelog fragment that will be added to the release notes when we cut the next release. This should be a single line or two at most.

**Review by jdrew82 — CHANGES_REQUESTED** [review]

File: changes/634.fixed (hunk 1 above)
Comment (michalis1) [reply changes/634.fixed]: I’ve made the change. I hope it’s better now.

**Review by jdrew82 — APPROVED** [review]
LGTM!
